modules/post/windows/gather/enum_unattend.rb

```diff
@@ -138,8 +133,7 @@ def init_paths
 
     # If there is one for registry, we add it to the list too
     reg_path = get_registry_unattend_path
-    paths << reg_path unless reg_path.empty?
-
+    paths << reg_path unless reg_path&.empty? or reg_path.nil?
```

Comment:
If you use `blank?` you can actually remove the nil check. It's well suited for situations like this. For example:
```
msf6 > pry
[*] Starting Pry shell...
[*] You are in the "framework" object

[1] pry(#<Msf::Framework>)> nil.blank?
=> true
[2] pry(#<Msf::Framework>)> ''.blank?
=> true
[3] pry(#<Msf::Framework>)> 'test'.blank?
=> false
[4] pry(#<Msf::Framework>)> 
```

```suggestion
    paths << reg_path unless reg_path.blank?
```

Reply:
oh that's nice, I didn't knew about this :)